REST/PowerShell/Variables/ChangeVariableValueinVariableSet.ps1

```diff
@@ -0,0 +1,92 @@
+#Allows HTTPS connections
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+$VarSetName = "YourVariableSetName"           #The exact name of the Variable Set you are trying to change a variable within. Not needed if you know ID
+$VarSetID = ""                                #Leave blank if you dont know your variable set ID
+
+$VariableName = "YourVariableName"            #The name of the variable you are trying to change the value of
+$VariableValue = "YourValue"                  #New value of Variable
+$VariableEnvName = "YourEnvironment"          #Scope of a variable to a environment, leave blank to scope to all
```

Comment:
The comment here is confusing. It's not **scoping** the *variable* to an *environment*, it's actually looking for a *variable* that is **scoped** to the *environment*.